### PR TITLE
Add pinyin for some word with rarely-used characters and misread word

### PR DESCRIPTION
**Pull request by SleepyAsh0191**
~The PR is currently only a draft. The reason for open this PR is because I want to see if there will be lint errors every time I check, and I can also share my work progress.~

Now ready for review

### CODE REVIEW

**Comment by SleepyAsh0191** [comment]
@xicri @Bill-Haku I've done my PR, plz review it.

**Review by xicri — DISMISSED** [review]
Looks good! Thanks!

File: dataset/dictionary/locations.json5 [hunk 1]
```diff
@@ -421,13 +424,15 @@
...
     },
+    pinyins: [{ char: "珉", pron: "min2" }],
   },
   {
     en: "Luhua Pool",
     ja: "漉華の池",
     zhCN: "渌华池",
     pronunciationJa: "ろくかのいけ",
     tags: [ "liyue", "location" ],
+    pinyins: [{ char: "渌", pron: "lu2" }],
```
Comment (Bill-Haku) [inline dataset/dictionary/locations.json5:435]: should be lu4 @SleepyAsh0191

**Review by Bill-Haku — CHANGES_REQUESTED** [review]
@SleepyAsh0191 I found some mistakes. You may check them again.

**Review by Bill-Haku — APPROVED** [review]

File: dataset/dictionary/characters.json5 [hunk 2]
```diff
@@ -823,6 +825,7 @@
     zhCN: "辛焱",
     pronunciationJa: "しんえん",
     tags: [ "liyue", "character-main" ],
+    pinyins: [{ char: "焱", pron: "yan3" }],
```
[the remark this answers is not in the document]
Comment (SleepyAsh0191) [reply dataset/dictionary/characters.json5]: thx for correct it (copilot sucks)

**Review by xicri — APPROVED** [review]

**Comment by xicri** [comment]
Deployed, thanks!
It looks mostly working as expected!

The words are sorted by updated date on the index page, so you can see the words which @SleepyAsh0191 added pinyin today on the index page.

https://genshin-dictionary.com/zh-CN/

@SleepyAsh0191 I found some pinyins are not displayed properly.

https://genshin-dictionary.com/zh-CN/violet-court/
https://genshin-dictionary.com/zh-CN/tears-of-the-calamitous-god/
https://genshin-dictionary.com/zh-CN/vasoma-fruit/

I guess the values of `pinyins[].char` are not matched with the character in `zhCN`.
For example, 菫 in `pinyins[].char` is probably a Japanese character. The length of the lines at the bottom of the character are slightly different between Chinese and Japanese.

**Comment by Bill-Haku** [comment]
@xicri There's also some problem with this word

https://genshin-dictionary.com/zh-CN/gladiators-finale/

We added 3 pinyins for 3 characters, but only the last one is displayed

**Comment by xicri** [comment]
@Bill-Haku Thanks, I'll fix it on xicri/genshin-dictionary#89.

**Comment by xicri** [comment]
I will add validation to check if target characters of pinyin exists in `zhCN`. I opened #130 to handle this.

**Pull request merged**